src/neo4j-arc/graph-visualization/GraphVisualizer/Graph/Graph.tsx

```diff
@@ -156,6 +159,13 @@ export class Graph extends React.Component<GraphProps, GraphState> {
     if (assignVisElement) {
       assignVisElement(this.svgElement.current, this.visualization)
     }
+
+    if (initialZoomToFit) {
+      this.initialZoomToFitTimeout = setTimeout(() => {
+        this.visualization?.handleZoomToFit()
+        clearTimeout(this.initialZoomToFitTimeout)
+      }, 1000)
```

Comment:
Perhaps out of scope, but it'd be nice if we did the zoom after the simulation reached some amount of cooling of rather than a simple timeout. If you're not interested in doing that work, we in query can pick it up later :+1:

Reply:
@OskarDamkjaer - I'll look into it. Thanks for reminding!

Reply:
A callback is added after the simulation is finished. So there's no longer hard-coded timeout needed. It's pretty much looking like how Bloom handles the initial zooming.

Maybe trigger this method by waiting 100-200ms more after the simulation finishes will make the animation looks less rushing. WDYT?

Reply:
Sample screenshot:
https://user-images.githubusercontent.com/26452483/177558099-e5bb8422-ab1a-4a23-b4bb-b41639e5a6ee.mov

Reply:
I had a closer look and it seems that's not the case 🤔

Reply:
> It looks like the zoom to fit runs more than once per query?

@OskarDamkjaer - it only runs once after the simulation is done.